webapp/components/_new/generic/ResponsiveImage/ResponsiveImage.spec.js

```diff
@@ -0,0 +1,70 @@
+import { mount } from '@vue/test-utils'
+
+import ResponsiveImage from './ResponsiveImage'
+
+const localVue = global.localVue
+
+describe('ResponsiveImage.vue', () => {
+  const propsData = {}
+  let wrapper
+
+  const Wrapper = () => {
+    return mount(ResponsiveImage, { propsData, localVue })
+  }
+
+  describe('mount', () => {
+    describe('with empty image prop', () => {
+      it('renders nothing', () => {
+        wrapper = Wrapper()
+        expect(wrapper.html()).toEqual('')
+      })
+    })
+
+    describe('given `null` as image prop', () => {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-01T06:48:38Z" title="Wednesday, April 1st 2020, 8:48:38 am +02:00">Apr 1, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,70 @@
+import { mount } from '@vue/test-utils'
+
+import ResponsiveImage from './ResponsiveImage'
+
+const localVue = global.localVue
+
+describe('ResponsiveImage.vue', () => {
+  const propsData = {}
+  let wrapper
+
+  const Wrapper = () => {
+    return mount(ResponsiveImage, { propsData, localVue })
+  }
+
+  describe('mount', () => {
+    describe('with empty image prop', () => {
+      it('renders nothing', () => {
+        wrapper = Wrapper()
+        expect(wrapper.html()).toEqual('')
+      })
+    })
+
+    describe('givem `null` as image prop', () => {
```


```suggestion
    describe('given `null` as image prop', () => {
```